### DISCUSSION TITLE
Support descriptions of individual files within a dataset, including their individual licenses metadata

### DISCUSSION THREAD

**Discussion by codycooperross** [datacite/datacite-suggestions#118] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

In a repository, a dataset may consist of multiple files (data, code, documentation, participant information sheets, etc.). These have different sizes and formats, for example, but may also have different licenses. The DataCite Metadata Schema does not have a graceful way of handling this. 

### What solution might meet your needs?

Support for individual file metadata, including size, format, and license. 

### What alternatives have you tried or considered?

Serializing file descriptions in existing properties like `size`, `format`, and `rightsList` with free text descriptions pointing to the relevant files. For example,

```
"sizes": ["file1.txt = 27 KB", "file2.xls = 1 MB"],
"formats": ["file2.txt = text/plain", "file2.xls = application/vnd.ms-excel"],
"rightsList": [
    { "rights": "file1.txt is licensed under X", ... },
    { "rights": "file2.xls is licensed under Y", ... }
]
```

### Is there anything else you would like to share?

May inform conversations around a `distribution` property:

- https://github.com/datacite/datacite-suggestions/discussions/42

[Link for DataCite staff](https://app.frontapp.com/open/cnv_b8sf45x?key=kjx0KNs4C_DKLP_v1A4NHnimM2SJbC5S)

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [X] Harvesters who index DataCite DOIs and metadata
- [X] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by KellyStathis** [comment]
We’re looking to gather more info on this topic! I’ve updated this related discussion: [Add "distribution" information for direct file links #42](https://github.com/datacite/datacite-suggestions/discussions/42) including a link to a short survey.  Let us know if you have any questions.

**Comment by KellyStathis** [comment]
We've shared a new proposal for a "Distribution" property here: https://github.com/datacite/datacite-suggestions/discussions/211 which includes a way to describe individual files.
